Task05/Task05/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task05
+{
+    internal class Program
+    {
+        public static void Main()
+        {
+            Console.WriteLine(Math.Round( Square(2 + Square(3 + Square(4 + Square(5)))),3));
+        }
+
+        public static double Square(double x)
+        {
+            return Math.Sqrt(x);
+        }
```

Comment:
Нет смысла делать метод, который вызывает другой метод. Можно было просто написать
```suggestion
        public static void Main()
        {
            Console.WriteLine(Math.Round( Math.Sqrt(2 + Math.Sqrt(3 + Math.Sqrt(4 + Math.Sqrt(5)))),3));
        }
```